apps/.babelrc

```diff
@@ -2,8 +2,10 @@
   "presets": ["es2015", "react" ],
   "plugins": [
     "add-module-exports",
+    "syntax-async-functions",
```

Comment:
This plugin comes from the package [`babel-plugin-syntax-async-functions`](https://www.npmjs.com/package/babel-plugin-syntax-async-functions).  I think this is currently working because we do depend on this package via `@storybook/react`:

```
@storybook/react@3.2.11
  babel-preset-env@1.6.1
    babel-plugin-transform-async-to-generator@^6.22.0
      babel-plugin-syntax-async-functions@^6.8.0
```
Since we're now using it directly, we should add this as a direct dependency in our package.json so we don't accidentally break our build while upgrading storybook later.  Same with [`babel-plugin-transform-regenerator`](babel-plugin-transform-regenerator) which we also seem to get from storybook right now.

Reply:
Good idea!